index.html

```diff
@@ -33,5 +33,9 @@
     <div id="app"></div>
     <script type="text/javascript" src="/static/env.js"></script>
     <!-- built files will be auto injected -->
+    <noscript>
+      <h3>We're sorry but Tokend Admin panel requires a browser that supports JavaScript.</h3>
+      <strong>Your browser either does not support JavaScript, or it has JavaScript support disabled. If you want to correctly view this page, please upgrade your browser or enable JavaScript support.</strong>
+    </noscript>
```

Comment:
Please try making it a bit more user-friendly. Introduce some simple markup and maybe links to how enable javascript, or links to new browsers.

You should not use anything sophisticated.

You can use these pages as reference:
https://hackerone.com/browser-not-supported.html (ie not supported by us
https://photos.google.com/browser-not-supported/)